domain/servicetemplate/servicetemplate.go

```diff
@@ -72,7 +71,6 @@ func (a *ServiceTemplate) Equals(b *ServiceTemplate) bool {
 func (a *ServiceTemplate) Hash() (string, error) {
 	tpl := *a
 	tpl.ID = ""
-	tpl.DatabaseVersion = 0
```

Comment:
The hash is meant not to contain version values. Should primaryTerm and seqno be set to 0 here?

Reply:
yes it should, fixed